templates/filled_template.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Hello {{ name }}</title>
+    <title>Hello {{ yourname }}</title>
   </head>
   <body>
-    <h1>Hi, {{ name }}!</h1>
+    <h1>Hi, {{ yourname }}!</h1>
     {{ name }} said that their favorite flavor is {{ flavor }}
```

Comment:
You shouldn't have to edit this file (instead, you could use the parameter name as it appears here); but if you do, then you should change all 3 "{{ name }}"